app/ui.js

```diff
@@ -594,15 +592,15 @@ var UI = {
         if (!UI.controlbarDrag) {
             // The goal is to trigger on a certain physical width, the
             // devicePixelRatio brings us a bit closer but is not optimal.
-            var dragThreshold = 10 * (window.devicePixelRatio || 1);
-            var dragDistance = Math.abs(ptr.clientY - UI.controlbarMouseDownClientY);
+            const dragThreshold = 10 * (window.devicePixelRatio || 1);
+            const dragDistance = Math.abs(ptr.clientY - UI.controlbarMouseDownClientY);
 
             if (dragDistance < dragThreshold) return;
 
             UI.controlbarDrag = true;
         }
 
-        var eventY = ptr.clientY - UI.controlbarMouseDownOffsetY;
+        const eventY = ptr.clientY - UI.controlbarMouseDownOffsetY;
```

Comment:
I'm not a huge fan of using `const` in cases like these, but I suppose I can overlook it as part of the conversion from `var` and we'll sort things out as we go in the future.